### DISCUSSION TITLE
Unable to get past setting an IP during post install setup / The IP being routed by Linux is not the IP....

### DISCUSSION THREAD

**Discussion by boredchilada** [Security-Onion-Solutions/securityonion#12895] [2.4] [answered]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

32

### Storage for /

250

### Storage for /nsm

250

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I'm currently on proxmox 8.1.4 and im trying to install SO but im not able to get past the setup network portion of the installation due to this message.

The IP being routed by Linux is not the IP address assigned to the management interface (ens18).
This is not a supported configuration, please remediate and rerun setup.

I've been troubleshooting for about two days now, and nothing i do really works. I've tried setting the vms ip to static, nope. DHCP doesnt work either. I gave it a static address via my top level firewall. Still doesnt work.

At this point I dont really know whats going or why its not going forward.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by boredchilada** [comment]
currently my proxmox config is as so
<img width="456" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/82670806/15936a01-ed07-40b9-8aed-0a10253287d6">

**Reply by cm-ops** [reply]
Does the monitoring interface have an IP assigned to it?

**Reply by boredchilada** [reply]
Both interfaces on the SO VM inside proxmox receive an IP, 

management has an IP in the 10.0.0.1/24
monitor has an ip in the 10.99.40.1/24

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/82670806/691bb718-c644-4089-b384-b50ee58ddfa5)

heres the vm settings

<img width="337" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/82670806/ddac9b87-84aa-4bf0-a51f-744cabfba82c">

**Reply by boredchilada** [reply]
Do you mean the actual server interface that is going to be used for monitoring?

im currently working on converting to ovs on proxmox but currently no. I can include it into my bridge though?

 I'm currently working with proxmox community to get it to OVS properly so heres more context 
 
 Top level NGFW <-> dumb switch <-> Cisco switch 2960x <-> wifi router

Currently the proxmox server is connecting via one interface (eno3) to the cisco switch, the 2nd interface(enp5s0) connects back to the dumb switch. Something like this
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/82670806/5ec00f6e-4e24-4687-a4a8-344e7b14f8cf)

I'm aiming for a OVS config such as this one:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/82670806/f086ac6e-ae99-4d77-bbe4-5a6b8936d18c)

**Reply by boredchilada** [reply]
This is weird, its  only allowing to use one interface out of 3 , and only that one?

I currently have 3 and its only letting me choose the one the middle, so not net0 or net2 just net1...?
<img width="238" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/82670806/4bd98a2c-12ec-4aa1-80a3-bdfbf1045d5b">

<img width="257" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/82670806/58cfc849-a9a9-4c92-8c2f-d2279d5d7236">

**Reply by reyesj2** [reply, marked as answer]
Check your ```ip route``` for duplicate gateways
Your monitor interface should not have an ip address assigned it should get connected to a SPAN port or TAP to ingest traffic. You can also look at setting up ingesting traffic from a vnic in proxmox https://docs.securityonion.net/en/2.4/proxmox.html#virtual-nic

**Reply by boredchilada** [reply]
Ok, so i finally made sense of all this and just created a SPAN port on my switch of all my ports and plugged the monitor interface from proxmox which is passed through pci , everything works well and getting up to 60mb/s packet ingestion at times:
<img width="132" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/82670806/78d498c4-6e83-49b8-b776-6f33e534c9e9">
